tests/nnstreamer_edge/edge/unittest_edge.cc

```diff
@@ -358,6 +359,169 @@ TEST (edgeSinkSrc, runNormalAitt)
   g_free (sink_pipeline);
 }
 #endif
+
+/**
+ * @brief Test for edgesink custom connection.
+ */
+TEST (edgeCustom, sinkNormal)
+{
+  /** @todo TDD: Enable this test later. */
+  GTEST_SKIP ();
+
+  gchar *pipeline;
+  GstElement *gstpipe;
+
+  /* Create a nnstreamer pipeline */
+  pipeline = g_strdup_printf (
+      "gst-launch-1.0 videotestsrc ! videoconvert ! videoscale ! "
+      "video/x-raw,width=320,height=240,format=RGB,framerate=10/1 ! "
+      "tensor_converter ! edgesink connect-type=CUSTOM custom-lib=%s name=sinkx port=0",
+      CUSTOM_LIB_PATH);
+  gstpipe = gst_parse_launch (pipeline, NULL);
+  EXPECT_NE (gstpipe, nullptr);
```

Comment:
Well written TC. It seems necessary to initialize ```gstpipe``` to nullptr for nullptr comparison.

Reply:
Updated. Thanks!

Reply:
Thank you.